test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorTest.cs

```diff
@@ -220,55 +220,6 @@ public void ValidateReturnsMemberNameIfItIsDifferentFromDisplayName()
 #endif
 
         [Fact]
-        public void GetClientValidationRules_ReturnsEmptyRuleSet()
```

Comment:
where is `GetErrorMessage()` checked?  what confirms that the client validator provider successfully returns an empty rule set for an unknown `ValidationAttribute`?  what confirms that provider handles the `IClientModelValidator` case in general (not just for `[Remote]`)?